test/compare_sgemm_sbgemm.c

```diff
@@ -76,9 +76,9 @@ float16to32 (bfloat16_bits f16)
 int
 main (int argc, char *argv[])
 {
-  int m, n, k;
+  blasint m, n, k;
   int i, j, l;
-  int x;
+  blasint x;
```

Comment:
Can you also please remove line#129 `for (l = 0; l < k; l++)` where we compare CC & DD.

Reply:
Oh, right, this loop does not really make sense - and same for line 115 I guess :)